package.json

```diff
@@ -49,7 +49,7 @@
     "coveralls": "^2.11.4",
     "eslint": "^4.1.1",
     "istanbul": "^0.4.0",
-    "jscs": "^3.0.0",
+    "jscs": "^2.11.0",
```

Comment:
3.x seems to hang freeze for some reason. Should think about moving to eslint for style linting too at some point.

Reply:
:+1: for moving to eslint, jscs is no longer supported. But that is outside the scope of this PR :-)